template/.claude/commands/simplify.md

```
@@ -0,0 +1,84 @@
+---
+description: Reduce code complexity while keeping tests green
+argument-hint: [file, function, or area to simplify]
+---
+
+## General Guidelines
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)